enrollment-server-onboarding/pom.xml

```
@@ -278,6 +270,27 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>public-repository</id>
+            <activation>
+                <property>
+                    <name>!useInternalRepo</name>
+                </property>
+            </activation>
+            <properties>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-deploy-plugin</artifactId>
+                        <configuration>
+                            <skip>true</skip>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
```

It is possible to configure plugin just via properties, it does not have to be explicitly declared. https://maven.apache.org/plugins/maven-deploy-plugin/deploy-mojo.html#skip
```suggestion
            <properties>
                <maven.deploy.skip>true</maven.deploy.skip>
            </properties>
```

---

Of course, consistency is important. This was just a hint, how it is possible to write it differently.